pyvista/core/objects.py

```diff
@@ -357,23 +357,126 @@ def get_data_range(self, arr=None, preference='row'):
 
 
 class Texture(_vtk.vtkTexture, DataObject):
-    """A helper class for vtkTextures."""
+    """A helper class for vtkTextures.
 
-    def __init__(self, *args, **kwargs):
+    Parameters
+    ----------
+    uinput : str, vtkImageData, vtkTexture, list, optional
+        Filename, ``vtkImagedata``, ``vtkTexture``, or a list of images to
+        create a cubemap. If a list of images. Must be of the same size and in
+        the following order:
+
+        * -X
+        * -Y
+        * -Z
+        * +X
+        * +Y
+        * +Z
+
+    **kwargs : dict, optional
+        Optional arguments when reading from a file.
+
+    Examples
+    --------
+    Load a texture from file. Files should be "image" or "image-like" files.
+
+    >>> import os
+    >>> import pyvista
+    >>> from pyvista import examples
+    >>> path = examples.download_masonry_texture(load=False)
+    >>> os.path.basename(path)
+    'masonry.bmp'
+    >>> texture = pyvista.Texture(path)
+    >>> texture  # doctest:+SKIP
+    Texture (0x7f3131807fa0)
+      Components:      3
+      Cube Map:        False
+      Dimensions:      256, 256
+
+    Create a texture from a RGBA array. Note how this is colored per "point"
+    rather than per "pixel".
+
+    >>> import numpy as np
+    >>> arr = np.array(
+    ...     [
+    ...         [255, 255, 255],
+    ...         [255, 0, 0],
+    ...         [0, 255, 0],
+    ...         [0, 0, 255],
+    ...     ], dtype=np.uint8
+    ... )
+    >>> arr = arr.reshape((2, 2, 3))
+    >>> texture = pyvista.Texture(arr)
+    >>> texture.plot()
+
+    Create a cubemap from 6 textures.
+
+    >>> nx = examples.download_sky(direction='nx')  # doctest:+SKIP
+    >>> ny = examples.download_sky(direction='ny')  # doctest:+SKIP
+    >>> nz = examples.download_sky(direction='nz')  # doctest:+SKIP
+    >>> px = examples.download_sky(direction='px')  # doctest:+SKIP
+    >>> py = examples.download_sky(direction='py')  # doctest:+SKIP
+    >>> pz = examples.download_sky(direction='pz')  # doctest:+SKIP
+    >>> texture = pyvista.Texture([nx, ny, nz, px, py, pz])  # doctest:+SKIP
```

Comment:
If you run this test and rotate the view around you'll see that the images don't line up. So the correct order is indeed
```suggestion
    >>> texture = pyvista.Texture([px, nx, py, ny, pz, nz])  # doctest:+SKIP
```